randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1050,7 +1055,10 @@ Extra - minimap_data: [{'x': 69, 'y': 27}, {'x': 69, 'y': 28}, {'x': 70, 'y': 27
   > Dock to Lower Factory Save Station
       Trivial
   > Pickup (Missile Tank)
-      Speed Booster and Shinesparking Tricks (Intermediate)
+      All of the following:
+          Speed Booster and Shinesparking Tricks (Beginner)
+          # Charge speedbooster
+          Movement (Intermediate) or Shinesparking Tricks (Beginner) or Disabled Entrance Rando
```

Comment:
Not sure what the intent of the second `Shinespark (Beginner)` is as it will always be satisfied due to the all of the following above.

Reply:
was supposed to be a short charge, whoops.